tests/backup/backup_test.go

```diff
@@ -2661,6 +2663,244 @@ var _ = Describe("{BackupAlternatingBetweenLockedAndUnlockedBuckets}", func() {
+				backupSuccessCheck := func() (interface{}, bool, error) {
+					backupDriver := Inst().Backup
+					bkpUid, err := backupDriver.GetBackupUID(ctx, recentBackupName, orgID)
+					if err != nil {
+						return "", true, err
+					}
+					backupInspectRequest := &api.BackupInspectRequest{
+						Name:  recentBackupName,
+						Uid:   bkpUid,
+						OrgId: orgID,
+					}
+					resp, err := backupDriver.InspectBackup(ctx, backupInspectRequest)
+					if err != nil {
+						return "", true, fmt.Errorf("unable to fetch inspect backup response for %v", recentBackupName)
+					}
+					actual := resp.GetBackup().GetStatus().Status
+					expected := api.BackupInfo_StatusInfo_Success
+					if actual != expected {
+						return "", true, fmt.Errorf("backup status for [%s] expected was [%s] but got [%s]", recentBackupName, expected, actual)
+					}
+					return "", false, nil
+				}
+				_, err = task.DoRetryWithTimeout(backupSuccessCheck, 10*time.Minute, 30*time.Second)
+				log.FailOnError(err, "Backup with name %s was not successful", recentBackupName)
+				log.InfoD("New backup - %s is successful from schedule backup ", recentBackupName)
+			}
+			log.InfoD("All Accesses are toggled and operations are performed")
+
+		})
+
+	})
+	JustAfterEach(func() {
+		defer EndTorpedoTest()
+		log.InfoD("End test")
+		ctx, err := backup.GetAdminCtxFromSecret()
+		log.FailOnError(err, "Fetching px-central-admin ctx")
+
+		//Get scheduleBackupUid
+		log.Infof("Get scheduleBackupUid")
+		scheduleBackupUid, err := GetScheduleUID(backupName, orgID, ctx)
+		log.InfoD("scheduleBackupUid - %v", scheduleBackupUid)
+		log.FailOnError(err, "Error in fetching schedule UID for %v", backupName)
+
+		//Delete Schedule Backup-
+		log.Infof("Delete Schedule Backup-")
+		err = DeleteSchedule(backupName, scheduleBackupUid, periodicPolicyName, schPolicyUid, orgID)
+		log.FailOnError(err, "Error deleting Schedule backup %v", backupName)
+
+		//GetAll backups -
+		backupNames, err := GetAllBackupsAdmin()
+		log.FailOnError(err, "Fetching admin backups")
+
+		//Delete Backup
+		backupDriver := Inst().Backup
+		for _, backupName := range backupNames {
+			log.InfoD("Deleting backup - %v", backupName)
+			backupUID, err := backupDriver.GetBackupUID(ctx, backupName, orgID)
+			log.FailOnError(err, "Failed while trying to get backup UID for - %s", backupName)
+			log.Infof("About to delete backup - %s", backupName)
+			backupDeleteResponse, err := DeleteBackup(backupName, backupUID, orgID, ctx)
+			log.FailOnError(err, "Backup [%s] could not be deleted with delete response %s", backupName, backupDeleteResponse)
+		}
+
+		log.Infof("Deleting restore for user")
+		ctxNonAdmin, err := backup.GetNonAdminCtx(username, password)
+		log.FailOnError(err, "Fetching %s ctx", username)
+
+		for _, restore := range restoreNames {
+			err := DeleteRestore(restore, orgID, ctxNonAdmin)
+			log.InfoD("Deleting restore %v", restore)
+			log.FailOnError(err, "Deleting User Restore")
```

Comment:
Let's print the `log.InfoD()` after the `log.FailOnError()` or print it before calling `err := DeleteRestore(restore, orgID, ctxNonAdmin)` and also can we add more details on what user or restoreID or something in the message please